mk/spksrc.service.mk

```diff
@@ -196,10 +196,16 @@ ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
 $(DSM_SCRIPTS_DIR)/installer: $(SPKSRC_MK)spksrc.service.installer.dsm7
 	@$(dsm_script_copy)
 else
+ifeq ($(call version_lt, ${TCVERSION}, 6.0),1)
```

Comment:
I think this could be written more consistently e.g. as greater than or equal to 7.0; dsm7 stuff, else greater than or equal to 6.0; dsm6 stuff else dsm5 stuff. Or the reverse. But I dislike this version.

Reply:
@publicarray i have a question regarding your comment in the servive.mk for DSM7
`# STARTABLE needs to be yes, Resource linking and unlinking works on start and stop`

I wanted to fix this, as only `SPK_COMMANDS` was tested here and `SPK_USR_LOCAL_LINKS` was ignored.
But my findings are, that it is not needed to omit `ctl_stop=no` (i.e. use default `ctl_stop=yes`) for resource linker to work.
The packages with `ctl_stop=no` are started by the system at installation time (and on every restart) and are stopped at uninstall.
This would simplify the code as it will depend on DSM >= 6.1 only to use `ctl_stop=no` or `startable=no`.

Reply:
@publicarray the implementation of `ctl_stop=no` is in `spksrc.spk.mk` and not in `spksrc.service.mk` but your comment is in both.

Reply:
Thanks for the optimisation. The downside is that all CLI apps have `STARTABLE=no` set. I just tested it again and CLI packages (I use `zsh` for testing) that have `ctl_stop=no` will still work but the `start/stop` button is obviously missing. IMHO we should override and remove the `ctl_stop=no` if `SPK_COMMANDS` is set. `SPK_USR_LOCAL_LINKS` doesn't guarantee an executable but yes it should also apply then.